tools/c7n_gcp/c7n_gcp/filters/sccfindings.py

```diff
@@ -0,0 +1,84 @@
+# Copyright The Cloud Custodian Authors.
+# SPDX-License-Identifier: Apache-2.0
+"""
+Security Command Center Findings suppport for GCP resources
+"""
+from c7n.filters.core import ValueFilter
+from c7n.utils import local_session, type_schema
+from c7n_gcp.provider import resources as gcp_resources
+
+
+class SecurityComandCenterFindingsFilter(ValueFilter):
+    """Filters resources based on their Security Command Center (SCC) findings.
+
+    .. code-block:: yaml
+
+      - name: bucket-contains-high-finding
+        resource: gcp.bucket
+        filters:
+        - type: scc-findings
+          org: 11111111111111
+          key: "[].finding.severity"
```

Comment:
it feels like we should be processing per finding instead of doing so on the set to make matching a little easier, else every key has to be prefixed [].. doing that means we can also annotate the matched finding onto the resource.

Reply:
I was debating whether I should do that or not since the api call also returns a resource section in addition to the finding, but I agree it makes it much cleaner to have it just on the findings, which is the use case we want anyway.

```
        "finding":{
           ...
         },
        "resource": {
          "name": "//storage.googleapis.com/gcf-sources-2222222222222-us-east1",
          "projectName": "//cloudresourcemanager.googleapis.com/projects/2222222222222",
          "projectDisplayName": "cloud-custodian",
          "parentName": "//cloudresourcemanager.googleapis.com/projects/2222222222222",
          "parentDisplayName": "cloud-custodian"
        }